### PR TITLE
New authorizer concept

### PR DESCRIPTION
**Pull request by leleuj**

### CODE REVIEW

File: pac4j-core/src/main/java/org/pac4j/core/authorization/OrMultiAuthorizer.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * @param <U> the user profile
+ * @author Jerome Leleu
+ * @since 1.8.0
+ */
+public class OrMultiAuthorizer<U extends CommonProfile> implements Authorizer<U> {
+
+    protected final Authorizer<U>[] authorizers;
+
+    public OrMultiAuthorizer(final Authorizer<U>[] authorizers) {
+        this.authorizers = authorizers;
```
Comment (miremond) [inline pac4j-core/src/main/java/org/pac4j/core/authorization/OrMultiAuthorizer.java]: maybe you can copy the array to avoid modification from the outside

**Comment by miremond** [comment]
+1

File: pac4j-core/src/main/java/org/pac4j/core/authorization/OrMultiAuthorizer.java (hunk 1 above)
Comment (leleuj) [inline pac4j-core/src/main/java/org/pac4j/core/authorization/OrMultiAuthorizer.java]: You're right: that may be a problem. Though, if you take a look at the `HasAccessRoleAuthorizer`, the `Authorizer` classes are built internally. Maybe I should make this constructor `protected`. Will keep that somewhere in my mind ;-)

**Comment by jeroen4now** [comment]
+100

**Pull request merged**